scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -37,6 +37,7 @@ workflow GvsBulkIngestGenomes {
         Int? load_data_preemptible_override
         Int? load_data_maxretries_override
         # End GvsImportGenomes
+        String branch_name = "ah_var_store"
```

Comment:
Unrelated entirely to your work - but call_set_identifier (above) is unused. I'll give you a nickel if you remove it.

Reply:
Yeah I found there's some other unused stuff as well; I'll make an additional cleanup pass.